eureka-client/src/main/java/com/netflix/discovery/util/DeserializerStringCache.java

```diff
@@ -36,7 +36,6 @@
     }
 
     private static final Logger logger = LoggerFactory.getLogger(DeserializerStringCache.class);
-    private static final boolean logEnabled = logger.isTraceEnabled();
```

Comment:
This is a typo, the actual intent here is `debugLogEnabled =logger.isDebugEnabled(); `

Reply:
Done